website_snippet_anchor/static/src/js/anchor_option.js

```diff
@@ -5,20 +5,29 @@
 odoo.define("website_snippet_anchor.anchor_option", function (require) {
     var core = require("web.core"),
         options = require("web_editor.snippets.options"),
-        website = require("website.website"),
+        utils = require("website.utils"),
         sprintf = _.str.sprintf,
         _t = core._t;
 
     // Option to have anchors in snippets
     options.registry.anchor = options.Class.extend({
-        // Let user choose anchor name
-        anchor_ask: function (type, window_title) {
-            // Only react on clicks, not on resets or other events
-            if (type != "click") {
-                return $.Deferred().reject();
+
+        start: function() {
+            var res = this._super.apply(this, arguments);
+            return res;
+        },
+
+        anchor_ask: function(type) {
+            if(type !== false) {
```

Comment:
If I'm not wrong, this should be `if (type !== "click")`, because the 1st argument that gets in is always the type of event, which could be `click`, `reset` or `hover`. We want to react only on click.